include/aspect/simulator.h

```diff
@@ -469,6 +481,92 @@ namespace aspect
        */
       void postprocess ();
 
+      // TODO: Identify whether and how the following functions can be moved
+      // out of the Simulator class, or which should be integrated as core
+      // functionality
```

Comment:
Yes. I think it would be really nice if all of the VoF functionality could be separated into a class of its own, similar to the `FreeSurface` class, for example. We don't want to put every possible functionality into the `Simulator` class, as it would quickly become rather unreadable.